tasks/sep/030-revert-mt-cannon/README.md

```diff
@@ -0,0 +1,17 @@
+# ProxyAdminOwner - Set Dispute Game Implementation
+
+Status: DRAFT, NOT READY TO SIGN
+
+## Objective
+
+This task updates the fault dispute system for op-sepolia: 
+
+* Set implementation for game type 0 to 0xF3CcF0C4b51D42cFe6073F0278c19A8D1900e856 in `DisputeGameFactory` 0x05F9613aDB30026FFd634f38e5C4dFd30a197Fa1: `setImplementation(0, 0xF3CcF0C4b51D42cFe6073F0278c19A8D1900e856)`
+* Set implementation for game type 1 to 0xbbDBdfe37C02439764dE0e41C906e4396B5B3914 in `DisputeGameFactory` 0x05F9613aDB30026FFd634f38e5C4dFd30a197Fa1: `setImplementation(1, 0xbbDBdfe37C02439764dE0e41C906e4396B5B3914)`
+<!--NEXT TASK DESCRIPTION-->
+
+This switches sepolia back to singlethreaded cannon, undoing the upgrade to cannon-mt (025-mt-cannon task)
+
+### State Validations
```

Comment:
This README is missing a few sections compared to some other [examples](https://github.com/ethereum-optimism/superchain-ops/blob/d04c75e4793251296310fb29481b80302449fa0e/tasks/sep/026-fp-holocene-upgrade-fix/README.md#L1-L42) like: Pre-deployments, Simulation, Execution.

Reply:
The info on predeployed contracts in included in the validation page.  Not sure we need to repeat instruction on simulating in every task but added.  The details of the execution are already listed in the Objective.